app/src/layouts/AuthLayout.test.tsx

```diff
@@ -0,0 +1,23 @@
+import { render } from '@testing-library/react';
+import React from 'react';
+import AuthLayout from './AuthLayout';
+import { Router } from 'react-router-dom';
+import { createMemoryHistory } from 'history';
+
+const history = createMemoryHistory();
+
+describe('AuthLayout', () => {
+  it('renders correctly', () => {
```

Comment:
I don't think this test is doing what you expect because `context.ready` is falsey so it is not actually rendering the `props.children`, instead it is rendering a circular spinner. 

I would say, rename this test to say `renders spinner when context is not ready`
and then we can work on adding the other test case later